packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -362,6 +364,7 @@ export type CallAdapterClientState = {
     cameraStatus?: 'On' | 'Off';
     videoBackgroundImages?: VideoBackgroundImage[];
     selectedVideoBackgroundEffect?: SelectedVideoBackgroundEffect;
+    acceptedTransferCall?: CallCommon;
```

Comment:
1. acceptedTransferCall is the call object returned from TransferRequestedEventArgs.accept() from the trasnferRequested event. I need to store it to find the corresponding CallState from CallClientState to track its state(ie: Connected, Connecting, Ringing etc) for the transfer page later on.
2. I see. I thought it would be okay since [EnvironmentInfo](https://github.com/Azure/communication-ui-library/blob/main/packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts#L14) is directly from the calling SDK. Would changing it to [CallCommon](https://github.com/Azure/communication-ui-library/blob/main/packages/calling-stateful-client/src/index.ts#L11) exported by the stateful client be ok?

Reply:
When I'm connecting to the accepted transfer call, what is the status of the old call state? (Is it still `connected`? `disconnected`? something else?)

Reply:
The old call state is 'Connected' still. While the accepted transfer call, starts from 'None', then 'Connecting', 'Ringing' and then 'Connected'

Reply:
So I have two different call objects both with connected? Or the SDK keeps the same call but transfers it (so same CallId and everything?)

Reply:
There are two different call objects. The SDK will remove the old call once the accepted transfer call has state 'Connected'.